### DISCUSSION TITLE
how to divide the grain into two parts according personal need

### DISCUSSION THREAD

**Discussion by USTChd** [neperfepx/neper#340] [Q&A] [answered]
Dear all,
I have obtained a 2D polycrystalline model with Voronoi tessellations. And then I want to divide each grain into two parts: a thin ring along the grain boundary and the rest part of the grain. But I don't know what should do even if reading the manual. Could give me any suggestions on it? Thanks a lot.
Dong

**Comment by rquey** [comment, marked as answer]
This is not possible at the moment, see [related discussions](https://github.com/rquey/neper/discussions?discussions_q=core-shell).
